eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/CoreExtension.php

```diff
@@ -39,6 +39,6 @@ public function getName()
      */
     public function getGlobals()
     {
-        return array('ezpublish' => $this->globalHelper);
+        return ['ez_app' => $this->globalHelper];
```

Comment:
Rather `ezplatform`, it's about the Product.

Reply:
Symfony provides `app` global helper , ezPlatform provides `ez_app` . So, it is because of naming compatibility with symfony.

Reply:
Still -1 from my side

Reply:
In most cases (with exceptions of course, but this is not the case here) renaming `ezpublish` to `ezplatform` is recommended.